examples/core/texture-3d/app.js

```diff
@@ -158,7 +158,9 @@ export default class AppAnimationLoop extends AnimationLoop {
   }
 
   onFinalize({gl, cloud}) {
-    cloud.delete();
+    if (cloud) {
```

Comment:
Why do we need the if statement? finalize should never be called if initialize didn't complete.

Reply:
The initialize/finalize logic needs to be rethought. I believe `onInitialize` is called once the first time `start` is called, while `onFinalize` is called everything `stop` is called.

Reply:
Its tracked here: https://github.com/uber/luma.gl/issues/1068